### ISSUE TITLE
aarch64 syntax error

### ISSUE THREAD

**Issue by cwnolt** [bigtreetech/sunxi-tools#1]
I have installed sunxi-tools on ubuntu and when I try to run sunxi-fel-aarch64 I get : ./sunxi-fel-aarch64: 2: Syntax error: "(" unexpected.
I have tried downloading everything a few times and it does the same thing every time.

**Comment by StarkRG** [comment]
That's because they've provided an ARM executable when I'm pretty sure they meant to provide an amd64 executable.

**Comment by treaves** [comment]
Yes, @bigtreetech , can we please get an Intel version?  @cwnolt did you get this?

**Comment by treaves** [comment]
Okay, ignore their filenames: the aarch64 is ARM 64 bit, the ARM is Intel 64 bit.
